### DISCUSSION TITLE
Is there a vim shortcut that creates nvim-surround's function call shortcut?

### DISCUSSION THREAD

**Discussion by ghost** [kylechui/nvim-surround#346] [Q&A] [answered]
I like nvim-surround's delete function call operation (dsf); it's extremely handy. It's not built in to VS Code's vim plugin, though. Interestingly, I'm able to *create* function calls with that plugin (e.g. `ysiwfHello` turns `foo` to `Hello(foo)`) but it doesn't accept `f` as a surround selector so I can't quickly remove a surrounding function call. How could I implement this functionality with a custom shortcut using vim commands?

**Comment by kylechui** [comment, marked as answer]
There's no real "easy" way to do it with a Vim emulation; I might suggest instead using vscode-neovim, which would allow you to use an init.lua.

**Reply by ghost** [reply]
Thanks! I was not aware of that plugin. Seems to work just fine for me (only downside is that I have to start over again introducing the functionality I want, though, lol.)
